### PR TITLE
Edit readme and add installation steps

### PR DESCRIPTION
**Pull request by cesswairimu**
#### Fixes issue #22 

- Add a project description
- Add installations notes

- [x] Tests all green tests and all builds passes :green_heart:
- [x] PR well-named, issue linked :paperclip: and no merge conflicts.
- [x] Request review from other contributors

### CODE REVIEW

File: Installations.md [hunk 1]
```diff
@@ -0,0 +1,121 @@
...
+#### Updating Rails
+If you need to update rails, you can run `bundle update rails`.  If you run into problems with rails and `libv8` on OS X, try this:
+```shell
+   gem uninstall libv8
+   brew install v8
+   gem install therubyracer
+   gem install libv8 -v '3.16.14.3' -- --with-system-v8
+```
+    
+### Step 2: Install javascript dependencies using `yarn`
```
Comment (aonomike) [inline Installations.md:80]: thanks @cesswairimu I am not sure we need to install js dependencies since we do not have asset files, what do you think?

**Comment by aonomike** [comment]
Hi @cesswairimu thanks so much for working on this. Do you think it is important to state that we are reusing the `schema.rb` for the current WebsiteOne project in the database setup section? We decided to do this with @ciremoussadia so we could just point our api to the current WSO live database and it would just pickup. I am not sure what @ciremoussadia  thinks about this.

**Comment by cesswairimu** [comment]
Yeah @aonomike I think it is crucial we add that...I will add that...do we have like a test/staging env that we shall use for local testing? Or how will that go? Thanks

**Comment by ciremoussadia** [comment]
@aonomike & @cesswairimu happy to read you guys. I think maybe we can have a session this week and talk about it. It's all about design choices IMO.

**Comment by aonomike** [comment]
Thanks @ciremoussadia and @cesswairimu , @cesswairimu about the test/staging, maybe we can discuss that with @tansaku ? I have reached out to him about us using some boxes on GCP but maybe he will prefer heroku or gigalixir, not sure 🤔

**Comment by pjbelo** [comment]
Hello @cesswairimu, @aonomike 
There seems to exist a typo and in some places appears `WebsiteOneBackendApiBackendApi` instead of `WebsiteOneBackendApi`
I did a search and there are 5 instances of this typo in [this commit](https://github.com/AgileVentures/WebsiteOneBackendApi/pull/37/commits/72552e0a8431f1bf0311a5214526b836b6fd941b).

**Comment by cesswairimu** [comment]
Wow...great catch @pjbelo :+1: will fix that. Thanks

**Comment by cesswairimu** [comment]
@aonomike @ciremoussadia @pjbelo I fixed the typo and added a note to say we are reusing Websiteone db...kindly review.

**Comment by cesswairimu** [comment]
@aonomike @ciremoussadia @pjbelo I fixed the typo and added a note to say we are reusing Websiteone db...kindly review.

**Review by aonomike — APPROVED** [review]
Looks good to me @cesswairimu. I will approve and merge. Maybe other additions and subtractions can be done on a separate PR?

**Pull request merged**